src/FlatSharp.Compiler/Visitors/FieldVisitor.cs

```diff
@@ -33,55 +33,30 @@ public override FieldDefinition VisitField_decl([NotNull] FlatBuffersParser.Fiel
         {
             this.definition.Name = context.IDENT().GetText();
 
-            ErrorContext.Current.WithScope(this.definition.Name, () =>
```

Comment:
Please keep the "this."; it's helpful to identify what's an instance field vs static field.